service/tests/service.test.js

```diff
@@ -0,0 +1,62 @@
+import { expect, test } from 'vitest'
+
+import dotenv from 'dotenv'
+
+dotenv.config();
```

Comment:
This would be better located in a vite setup file and not in the test, unless you want each test to be able to load a different .env file (but I don't think that's the case here).

Reply:
Like you said to me I tried to use the --env variable and get rid of the _dotenv_ dependency but didn't make it to work, I think it was for my nodeJs version. So I left all of that and didn't change it.